add a note about unread and mention syncs happening hourly (#7760)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/communicate-with-users-from-other-organizations.md

<sup>3</sup> Supported for 1:1 chat for Teams Only to Teams Only users from two different organizations. <br>
<sup>4</sup> By default, external participants can't see the phone numbers of dialed-in participants. If you want to maintain the privacy of these phone numbers, select **Tones** for **Entry/exit announcement type** (this prevents the numbers from being read out by Teams). To learn more, read [Turn on or off entry and exit announcements for meetings in Microsoft Teams](turn-on-or-off-entry-and-exit-announcements-for-meetings-in-teams.md). <br>
<sup>5</sup> Allowed by default, but can be turned off by the Teams admin

> <sup>6</sup> Cross-tenant unread message count and at-mention notifications are updated about once per hour in the Teams desktop and web clients. 

Reviewer: ```suggestion
<sup>6</sup> Cross-tenant unread message count and @mention notifications are updated approximately once per hour in the Teams desktop and web clients. 
```
